.github/ISSUE_TEMPLATE/documentation.yml

```diff
@@ -0,0 +1,50 @@
+name: Documentation errors
+description: Let us know about typos and goofs in our site, documentation or API documentation blocks
+title: "[Docs]: "
+labels: ["documentation", "api"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Use this form to report a mistake or discrepancy with the core Foundry software site,
+        documentation, or code documentation comments.
+  - type: dropdown
+    id: course-where
+    attributes:
+      label: Where did you find it?
+      options: 
+       - Main Foundry site (not package listings)
```

Comment:
```suggestion
       - Knowledge Base (https://foundryvtt.com/kb/)
```

Reply:
Slight disagree, though calling out the KB by name is one thing, there is the rest of the site, where there is a non-zero chance for typos. Meet in the middle?
```diff
-       - Main Foundry site (not package listings)
+       - Knowledge Base (https://foundryvtt.com/kb/)
+       - Other Foundry site page (not package listings)
 ```